closed/OpenJ9.gmk

```diff
@@ -219,9 +219,9 @@ endif
 
 # Adjust JITServer enablement flags.
 ifeq (true,$(OPENJ9_ENABLE_JITSERVER))
-  FEATURE_SED_SCRIPT += $(call SedEnable,build_jitserver)
+  FEATURE_SED_SCRIPT += $(call SedEnable,opt_jitserver)
 else
-  FEATURE_SED_SCRIPT += $(call SedDisable,build_jitserver)
+  FEATURE_SED_SCRIPT += $(call SedEnable,opt_jitserver)
```

Comment:
This should call `SedDisable`; once my local build completes. I'll open a pull request to fix it.

Reply:
Right, my bad :(

Reply:
Fixed via https://github.com/ibmruntimes/openj9-openjdk-jdk14/pull/18.